src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -590,20 +503,15 @@ private void InitEntryExecution(ExecutionEnvironment env, ulong gasLimit, IRelea
             IReleaseSpec spec)
         {
             var spentGas = gasLimit;
-            if (substate.IsError)
-            {
-                return spentGas;
-            }
-            
+            if (substate.IsError) return spentGas;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)